### DISCUSSION TITLE
Different elements : Responsive control for divider

### DISCUSSION THREAD

**Discussion by 79ho3ein** [elementor/elementor#24243] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Some elements such as mega menu, icon list and even WordPress menu have divider option. When we enable this option, it applies to all breakpoints. But it would be great if we could control **weight** and **height** in this option responsively.

### Describe the solution you'd like

Responsive control of weight and height options in the divider

![divider](https://github.com/elementor/elementor/assets/77163248/9249cd25-4a13-4d11-af5f-21b7e8a2d9e5)

**Comment by saawood13** [comment]
I have also come across this limitation, specifically hiding dividers on breakpoint drop down menus
